Makefile

```diff
@@ -1,12 +1,12 @@
-DB_URL=postgres://postgres:12345@localhost:5432/postgres?sslmode=disable
+DB_URL=postgres://postgres:12345@db:5432/postgres?sslmode=disable
 
 .PHONY: migrate-up migrate-down create-migration proto
 
 proto:
 	protoc --proto_path=protobuf --go_out=proto --go_opt=paths=source_relative --go-grpc_out=proto --go-grpc_opt=paths=source_relative protobuf/*.proto
 
 run:
-	JWT_TOKEN_SECRET=my_secret_key REST_PORT=8080 GRPC_PORT=50052 ADDRESS=0.0.0.0 go run cmd/main.go
+	go run cmd/main.go
```

Comment:
doesn't work without `.env` file loaded _before_. 
need to have `.env` file sourced before this action